tests/ui/typeck/issue-103899.rs

```diff
@@ -2,6 +2,7 @@
 // failure-status: 101
 // dont-check-compiler-stderr
 // known-bug: #103899
+// rustc-env:RUSTC_ICE=0
```

Comment:
Can we please run the UI test by default with RUSTC_ICE=0?

Reply:
Done